sdk/javascript/messaging/stream.js

```diff
@@ -94,7 +94,6 @@ class Stream {
   }
 
   send (type, content) {
-    console.log('Sending', Message.MessageType.stringValue(type))
```

Comment:
Can you confirm with @peterschwarz that these log statements are no longer useful?